src/platforms/apple/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -86,6 +86,38 @@ SentrySDK.start { options in
 [NSURLSession]: https://developer.apple.com/documentation/foundation/nsurlsession
 [NSURLConnection]: https://developer.apple.com/documentation/foundation/nsurlconnection
 
+## File I/O Instrumentation
+
+<Alert level="info" title="Important">
+
+This is an experimental feature and requires an opt-in. Experimental features are still a work-in-progress and may have bugs. We recognize the irony.
+
+</Alert>
+
+The Sentry SDK adds spans for file I/O operations to ongoing transactions bound to the Scope. Currently, the SDK supports operations with [NSData][NSData], but many other APIs like [NSFileManager][NSFileManager] use [NSData][NSData].
```

Comment:
I know it's not consistent across the board, but mostly we don't capitalize "Scope" in this context in the docs. 

```suggestion
The Sentry SDK adds spans for file I/O operations to ongoing transactions bound to the scope. Currently, the SDK supports operations with [NSData][NSData], but many other APIs like [NSFileManager][NSFileManager] use [NSData][NSData].
```